### PR TITLE
fix: updating translation issues

### PR DESCRIPTION
**Pull request by hugovms**
| Q             | A
| ------------- | ---
| Branch?       | 5.x for features / 4.4 or 5.1 for bug fixes 
| Bug fix?      | yes/no
| New feature?  | yes/no 
| Deprecations? | yes/no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you. 

Could I ask @adielcristo and @cenoura to review this? Maybe also @brunosinister could have a look?

File: src/Symfony/Component/Form/Resources/translations/validators.pt.xlf [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
             </trans-unit>
             <trans-unit id="29">
                 <source>The uploaded file was too large. Please try to upload a smaller file.</source>
-                <target>O arquivo enviado é muito grande. Por favor, tente enviar um ficheiro mais pequeno.</target>
+                <target>O arquivo enviado é muito grande. Por favor, tente enviar um arquivo menor.</target>
             </trans-unit>
             <trans-unit id="30">
                 <source>The CSRF token is invalid. Please try to resubmit the form.</source>
-                <target>O token CSRF é inválido. Por favor submeta o formulário novamente.</target>
+                <target>O token CSRF está inválido. Por favor tente enviar o formulário novamente.</target>
```
Comment (adielcristo) [inline src/Symfony/Component/Form/Resources/translations/validators.pt.xlf]:
```suggestion
                <target>O token CSRF está inválido. Por favor, tente enviar o formulário novamente.</target>
```

**Review by Nyholm — APPROVED** [review]
Thank you @hugovms and @adielcristo

**Comment by derrabus** [comment]
Thank you @hugovms.

**Review by brunosinister — COMMENTED** [review]
Great job.

**Pull request merged**